src/components/SelectionList/CheckboxListItem.js

```diff
@@ -53,29 +82,20 @@ function CheckboxListItem({item, isFocused = false, onSelectRow, onDismissError
                         />
                     )}
                 </View>
-                {Boolean(item.avatar) && (
-                    <Avatar
-                        containerStyles={styles.pl5}
-                        source={lodashGet(item, 'avatar.source', '')}
-                        name={lodashGet(item, 'avatar.name', item.text)}
-                        type={lodashGet(item, 'avatar.type', CONST.ICON_TYPE_AVATAR)}
-                    />
-                )}
-                <View style={[styles.flex1, styles.flexColumn, styles.justifyContentCenter, styles.alignItemsStart, styles.pl3, styles.optionRow]}>
-                    <Text
-                        style={[styles.optionDisplayName, isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, styles.sidebarLinkTextBold]}
-                        numberOfLines={1}
-                    >
-                        {item.text}
-                    </Text>
-                    {Boolean(item.alternateText) && (
-                        <Text
-                            style={[isFocused ? styles.sidebarLinkActiveText : styles.sidebarLinkText, styles.optionAlternateText, styles.textLabelSupporting]}
-                            numberOfLines={1}
+                {Boolean(item.avatar) &&
+                    (showTooltip ? (
+                        <UserDetailsTooltip
+                            accountID={item.accountID}
+                            shiftHorizontal={styles.pl5.paddingLeft / 2}
                         >
```

Comment:
@cristipaval @aldo-expensify 
I posted the same question on refactoring PR too, are we good with this?

The `SelectionList` is a generic component, `CheckBoxListItem` is one of the item type and has user-specific components making this not reusable.

Earlier [thread](https://github.com/Expensify/App/pull/22622#discussion_r1297701124)

Since the tooltip is specfic to Members page only it's better we keep it there.

Reply:
@Santhosh-Sellavel 

Currently all usages of `CheckboxListItem` are in `WorkspaceMembersPage` and `WorkspaceInvitePage`, and both are using the Tooltips, so it's a common rule for all. If you think well, this whole list item represents a user, and currently there is no case where it's used differently.

I think it doesn't make sense to add complexity here to handle cases we don't even know it's gonna exist. If more multiple selection lists come in and they are not meant to represent a user, we handle it at that time, because we will have more info on how it's gonna work.

Also, we don't need to be attached to names, this `CheckboxListItem` could easily be renamed to `UserListItem` if we wanted, and then we could create a new, simpler `CheckboxListItem` for a new case that arrives.

Reply:
@thiagobrez 
We could just go with `UserListItem` straight away instead as it just serves that purpose. When CheckboxListItem is needed we can implement it. 

cc: @cristipaval @aldo-expensify

Reply:
@Santhosh-Sellavel Renamed to `UserListItem`

Reply:
Thanks for handling this, I agree with the outcome of renaming this to `UserListItem` for now.